### PR TITLE
Adding release workflow to CI

### PR DESCRIPTION
**Pull request by ghomanyi**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -51,7 +61,17 @@ jobs:
       - run:
           name: Run Blackduck Detect
           command: |
-            bash <(curl -s https://raw.githubusercontent.com/DACH-NY/security-blackduck/master/synopsys-detect) ci-build digitalasset_ex-structured-products master --logging.level.com.synopsys.integration=DEBUG
+            bash <(curl -s https://raw.githubusercontent.com/DACH-NY/security-blackduck/master/synopsys-detect) ci-build digitalasset_ex-structured-products master --logging.level.com.synopsys.integration=DEBUG --detect.notices.report=true
+      - run:
+          command: cp digitalasset_ex_structured_products_master_Black_Duck_Notices_Report.txt target/NOTICE
+      - persist_to_workspace:
+          root: target
+          paths:
```
Comment (bartfaitamas) [inline .circleci/config.yml]: Don't we want a lot more files there? LICENSE, README.md, conf.files?

Comment (ghomanyi) [reply .circleci/config.yml]: do we? I don't really know what we _have to_ package. All _I_ need are the jars and dar.

File: scripts/release.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+ download hub
```
Comment (bartfaitamas) [inline scripts/release.sh]: This supposed to be a comment?

File: scripts/release.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+export GITHUB_TOKEN="${GITHUB_REFAPP_MACHINE_TOKEN}"
+version=$(git describe --abbrev=0 --tags)
+number=$(echo $version | cut -c 2-)
+
+echo "testing if tag is on master"
+if git merge-base --is-ancestor $(git rev-parse $version) $(git rev-parse master)
+then
+    echo "collecting package"
+    fullname="structured-products-$number"
+    mkdir -p "/tmp/workspacetarget/$fullname/bin"
```
Comment (bartfaitamas) [inline scripts/release.sh]: why `/tmp/workspacetarget` , now `/tmp/workspace/target`

Comment (ghomanyi) [reply scripts/release.sh]: because later on there is a `cp -r /tmp/workspace/* ...` which would pick this up

**Pull request merged**